src/sqs/backfill-error-handler.ts

```diff
@@ -0,0 +1,44 @@
+import { BackfillMessagePayload, ErrorHandler, ErrorHandlingResult, SQSMessageContext } from "~/src/sqs/sqs.types";
+import { markCurrentTaskAsFailedAndContinue, TaskError } from "~/src/sync/installation";
```

Comment:
Should we move `markCurrentTaskAsFailedAndContinue` into this file instead? 
I don't think `sync/installation` is using it anymore?

Reply:
It uses subscription and reposyncstate, don't want to move these two over here

Reply:
Okay, I can accept this, but still want to point out that `markCurrentTaskAsFailedAndContinue` is ALL ABOUT handling errors, which is exactly what this file is for. 
But never mind, I am fine with keeping this inside the origin file.

Reply:
I think about sync/installation as a class, therefore all manipulations with the state should be encapsulated inside it. What is exported is a public interface. Moving the internals outside means to break encapsulation. IMHO